src/mmkv/mmkv.ts

```diff
@@ -4,12 +4,25 @@ import { Synchronizer } from '../types'
 const mmkv = new MMKV()
 const pendingChange = new Map<string, true>()
 
+type StorageOptions<T> = {
+    localStorageKey?: string
```

Comment:
mmkvKey or smth like that

Reply:
you can event move this whole type to types.ts and name this key just storageKey